acme/acme/challenges_test.py

```diff
@@ -359,11 +359,14 @@ def test_simple_verify(self, mock_verify_cert):
             cert=mock.sentinel.cert))
         mock_verify_cert.assert_called_once_with(self.msg, mock.sentinel.cert)
 
-    def test_simple_verify_false_on_probe_error(self):
+    @mock.patch('acme.challenges.socket.gethostbyname')
+    def test_simple_verify_false_on_probe_error(self, mock_gethostbyname):
         chall = mock.Mock()
         chall.probe_cert.side_effect = errors.Error
+
+        mock_gethostbyname.return_value = '127.0.0.2'
         self.assertFalse(self.msg.simple_verify(
-            self.chall, self.domain, self.key.public_key()))
+            self.chall, self.domain, self.key.public_key(), host="127.0.0.1"))
```

Comment:
In summary the only change in this PR should be: `self.chall` to `chall`. `host=...` should also be removed.